nimbus.go

```diff
@@ -3,15 +3,33 @@
 package main
 
 import (
-	statusnim "github.com/status-im/status-nim"
-	"runtime"
+	"crypto/ecdsa"
+
+	"github.com/jroimartin/gocui"
+	nimbusbridge "github.com/status-im/status-protocol-go/bridge/nimbus"
+	whispertypes "github.com/status-im/status-protocol-go/transport/whisper/types"
 )
 
 func init() {
-	runtime.LockOSThread()
+	nimbusbridge.Init()
+}
+
+func startNimbus(privateKey *ecdsa.PrivateKey, listenAddr string, staging bool) error {
+	return nimbusbridge.StartNimbus(privateKey, listenAddr, staging)
+}
+
+func schedulePoll(g *gocui.Gui, pollFunc func()) {
+	if pollFunc == nil {
+		return
+	}
+
+	g.Update(func(g *gocui.Gui) error {
+		pollFunc()
+		schedulePoll(g, pollFunc)
```

Comment:
Shouldn't it use some sleeping? It resembles `while(true)` loop now and will trigger UI update very often.

Reply:
Good point. On the other hand, I really dislike sleeps on the UI thread, so I'd rather implement something which schedules a regular operation (e.g. every 50ms) so that the main thread isn't blocked.

Reply:
@adambabik please re-review this part of the code. I made it so that it is called every 50ms from a background goroutine.